### PR TITLE
Output more info in pending-publish CLI

### PR DESCRIPTION
**Pull request by dirkmc**
```
$ ./lotus-miner storage-deals pending-publish
Publish period:             1h0m0s (ends in 55m11s)
First deal queued at:       2021-02-08 11:08:39.222417 +0100 CET
Deals will be published at: 2021-02-08 12:08:39.222417 +0100 CET
2 deals queued to be published:
ProposalCID                                                  Client                                                                                  Size
bafyreickthq7mv2lr2lyeigyle2gpihw63cga5rmiecnigic6zepq4vjja  t3udoamurqpl2e6yac6bvm2dzq2cxpu26rurt2vph3xowcegplslxx6zkfcw2u4l7k2lpo4n25lbr3diuqxwcq  2KiB
bafyreigmvme6a3eoctzhpjt3dc7jexv2qacn25mk6dhwyezcizamfcw5oa  t3udoamurqpl2e6yac6bvm2dzq2cxpu26rurt2vph3xowcegplslxx6zkfcw2u4l7k2lpo4n25lbr3diuqxwcq  2KiB

$ ./lotus-miner storage-deals pending-publish --publish-now
triggered deal publishing

$ ./lotus-miner storage-deals pending-publish
No deals queued to be published
```

### CODE REVIEW

File: cmd/lotus-storage-miner/market.go [hunk 1]
```diff
@@ -858,18 +858,25 @@ var dealsPendingPublish = &cli.Command{
 			return xerrors.Errorf("getting pending deals: %w", err)
 		}
 
-		w := tabwriter.NewWriter(os.Stdout, 2, 4, 2, ' ', 0)
-		_, _ = fmt.Fprintf(w, "ProposalCID\tClient\tSize\n")
+		if len(pending.Deals) > 0 {
+			w := tabwriter.NewWriter(os.Stdout, 2, 4, 2, ' ', 0)
+			_, _ = fmt.Fprintf(w, "Publish period:             %s\n", pending.PublishPeriod)
```
Comment (magik6k) [inline cmd/lotus-storage-miner/market.go:863]: Maybe also add `(ends in %s)` with the time from now to the period end

**Review by magik6k — COMMENTED** [review]
One UX suggestion

File: cmd/lotus-storage-miner/market.go (hunk 1 above)
Comment (dirkmc) [reply cmd/lotus-storage-miner/market.go]: 👍  added

Comment (dirkmc) [reply cmd/lotus-storage-miner/market.go]:
```
$ ./lotus-miner storage-deals pending-publish
Publish period:             1h0m0s (ends in 55m11s)
First deal queued at:       2021-02-08 11:08:39.222417 +0100 CET
Deals will be published at: 2021-02-08 12:08:39.222417 +0100 CET
...
```

**Review by magik6k — APPROVED** [review]

**Pull request merged**